razor-chase/benches/perf_test.rs

```diff
@@ -56,10 +56,7 @@ fn time_basic(theories: &Vec<Theory>) {
 }
 
 fn solve_basic(theory: &Theory) -> Vec<r#impl::basic::Model> {
-    let sequents: Vec<r#impl::basic::Sequent> = theory
-        .formulae
-        .iter()
-        .map(|f| f.into()).collect();
+    let sequents: Vec<r#impl::basic::Sequent> = theory.formulae.iter().map(|f| f.into()).collect();
```

Comment:
@Atul9 @KiChjang I’m not a huge fan of these long chains in one line. Is it unidiomatic to break them in multiple lines in Rust?

Reply:
It is idiomatic, it's just that your existing way isn't due to the last `.collect()` call:
```suggestion
    let sequents: Vec<r#impl::basic::Sequent> = theory
        .formulae
        .iter()
        .map(|f| f.into())
        .collect();
```

alternatively, and I like this style more:

```suggestion
    let sequents = theory
        .formulae
        .iter()
        .map(|f| f.into())
        .collect::<Vec<r#impl::basic::Sequent>>();
```

Reply:
ah I see! 👍 I didn't know the last syntax but I like it better too. It's interesting because type ascription is unstable but it looks like it's not the case here.